### ISSUE TITLE
Consolidate review deployments into one place

### ISSUE THREAD

**Issue by hdrury1** [elifesciences/elife-xpub#266]
**User Story:**

_As a member of eLife staff I want a single link in order to see all of the project deveopments in one place._

**Product requirements/Acceptance criteria:**

Functional:

Demo site should show all of the visual changes that have been made (and will be made).
Demo site should encompass all of the functionality changes that have been made (and will be made).
Demo site should display date of last update and confirmation that this is the most recent deployment.

**Comment by jsms90** [comment]
@hdrury1 Tam has just shown me this.

Follow this link, to see our environments in gitlab: https://gitlab.com/elifesciences/elife-xpub/environments.

We have 2 environments: "staging" & "demo"
![image](https://user-images.githubusercontent.com/15656538/42085487-77fc8f4a-7b88-11e8-8188-5e2a6c05ec58.png)


Staging - corresponds directly to what is in our `develop` branch. So click on the little "open" icon (on the right hand side), to see everything that has been merged via our PRs:
![image](https://user-images.githubusercontent.com/15656538/42085499-80d3a2fc-7b88-11e8-9b16-858a310096d4.png)

But, as @tamlyn said, we want to demo what we've done via a different environment, so we can keep pushing to `develop` (staging environment) & not break whatever you're demo-ing while you're demo-ing it :+1:  Hence, getting the "demo" environment in line with "staging" has to be done manually.

To do this, click on the "deploy to.." icon (on the RHS of the "staging" environment
![image](https://user-images.githubusercontent.com/15656538/42085596-cd2da7c4-7b88-11e8-8d0f-867a9fa54d24.png)

@tamlyn has literally just done this.

So now, if you click on the "open" icon on the RHS of the demo environment, you should see our app, with all the currently merged changes. All ready for you to show to Nick & Juliano :tada: 

Hope that helps

**Comment by hdrury1** [comment]
Yes it helps. Except I can't get past the first screen of the wizard (Next doesn't take me to the next page).

**Comment by jsms90** [comment]
@hdrury1 This pre-fill issue isn't happening on the demo site as far as I can see: https://xpub-elife-demo.gateway.xpub.semioticsquares.com/

@tamlyn has raised a PR to fix the 'Next' problem.

Can we close this issue? All it amounts to is saying "we need all the PRs merged" & isn't a separate body of work that needs to be done, unless I'm misunderstanding something about this "second demo site" that we have?

**Comment by hdrury1** [comment]
Pre-filling still a problem for me. Do I need to clear something- is this information being recovered from a time when I previously accessed the site?

**Comment by tamlyn** [comment]
Yes, the submission form shows your in progress submission if there is one. This is a decision we made in the vacuum of product requirements before you joined. Very happy to revisit it. Once you click submit on the final page of the wizard, you can come back and start a new submission. However note that this functionality is currently broken but fixed in #274.

**Comment by hdrury1** [comment]
Okay, that makes sense thanks @tamlyn, and progress should indeed be saved. Just waiting on the 'Next' button to be fixed and we should be able to share the demo site out.

**Comment by diversemix** [comment]
Ok. Let's be clear that everything that is on "semioticsquares.com" is not managed by eLife and we are moving away from this. The aim is to deploy onto AWS using the standard "builder" repo. 

As part of this ticket I have updated the docs and added the version to the footer.

Also I suggest we create some extra tickets ... see #245 - as  there is a number of clean up tasks that need to be performed:

1) Removal of https://xpub-elife-demo.gateway.xpub.semioticsquares.com/ - this is confusing issues as we have an eLife managed site https://demo--xpub.elifesciences.org

2) Re-organization of the dockerfiles. We currently have a development container - which just mounts the local disk as a volume - in fact the `Dockerfile-development` is just plain confusing as the files that are copied are just re-mounted over. Not sure how this was designed to have worked?

3) Remove the "yarn install" from the command used in "docker run" - this breaks the immutable infrastructure principle. 

4) Create a "test container" which will be designed to run against the elife-xpub container. Add to this container the extra stuff like chrome that is currently added to the xpub container. Change the e2e tests to spin this up and run against an immutable xpub container that is built.

As can be seen there is quite a number of tasks to get the deployment sorted. I talked with @BlueReZZ on Friday and he is happy we can talk to @giorgiosironi for ongoing help :+1: 

Happy to leave the k8s builds for the branches we are working on - but not sure we should be pushing these up to docker-hub. We do plan to move to k8s in the longer term - but either way we need to make sure we are testing and deploying the right thing (currently we test a different build to what we deploy - given 2 separate yarn installs)

**Comment by g-sam** [comment]
This issue seems to have spun off onto a different topic. Should we close?
Having two lists of deployments is unavoidable atm as the GitLab one is dynamically updated on Gitlab, while the eLife envs are in the docs.

**Comment by diversemix** [comment]
Thanks @g-sam - I will close this, @hdrury1 and I will work to raise tickets as and when they come. I had a very useful chat with @giorgiosironi about how they do their container management and testing so it will be good to stick with a similar paradigm.

**Comment by diversemix** [comment]
I will leave this open until the footer change get merged

**Comment by diversemix** [comment]
Version is now in the footer #278

**Comment by hdrury1** [comment]
@diversemix Could you confirm the address for the 'one place' I can direct folks to?

**Comment by diversemix** [comment]
I was going to say: https://demo--xpub.elifesciences.org/ ... but it seems as though this is still not automatically building from the latest docker container ... let me check
